packages/SwingSet/src/controller.js

```diff
@@ -10,11 +10,11 @@ import { assert } from '@agoric/assert';
 
 import makeDefaultEvaluateOptions from '@agoric/default-evaluate-options';
 import bundleSource from '@agoric/bundle-source';
-import { makeMemorySwingStore } from '@agoric/swing-store-simple';
 import {
   SES1ReplaceGlobalMeter,
   SES1TameMeteringShim,
 } from '@agoric/tame-metering';
+import { makeSwingStore } from '@agoric/swing-store-simple';
```

Comment:
Any opinions about making this the default export? To enable `import makeSwingStore from '@agoric/swing-store-simple'`, since the module isn't really exporting anything else (and could always use non-default exports if it needed to).

I suppose a counter-argument is that the name to which one binds the default export is usually recommended to be the same name as the package (else one might do something confusing and completely wrong like `import bundleSource from '@agoric/swing-store-simple'`), and in this case we don't want the local name to be `swingStoreSimple`, because the whole point is to remove the `simple` from it.

Reply:
I think I find the counter argument more persuasive.  Also, swing-store-simple does export a  couple of other things.